client/src/common/components/Members/components/MemberDetailsHeader.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Avatar from 'common/components/Avatar';
+import MemberRole from 'common/components/Members/components/MemberRole';
```

Comment:
I'm not sure if we need so many times `components`. Maybe if we have `Members` folder then all jsx files should be just inside it without any additional `components` folder?